src/theme/Colors.js

```diff
@@ -12,7 +12,7 @@ export function getColors() {
       light: grey[300],
       medium: '#C1C1C1',
       dark: '#444444',
-      darkest: grey[900],
```

Comment:
Haven't looked at these in a while, but I'm reminded that their palettes are supposed to be balanced, so arguably we should just replace all hard-coded numbers with grey[] selectors.

Would the greys here work for you?

https://mui.com/material-ui/customization/color/#color-palette

Besdies grey, only exception I'd make now is keeping our logo green on purpose to make it stick out of pallette :)

Reply:
Yeah totally